src/SignTool/SignTool/SignTool.SignToolBase.cs

```diff
@@ -46,6 +46,11 @@ public void Sign(IEnumerable<FileSignInfo> filesToSign, TextWriter textWriter)
                 string fileName = MSBuildPath;
                 var commandLine = $@"/v:m ""{buildFilePath}""";
 
+                if (Path.GetFileNameWithoutExtension(fileName).Equals("dotnet", StringComparison.OrdinalIgnoreCase))
+                {
+                    commandLine = $"build {commandLine}";
+                }
```

Comment:
Just thought through this and not sure this is going to work. The signing tool uses MSBuild tasks IIRC and it's unlikely they've been ported to target the non-desktop MSBuild.

Reply:
Yeah, I talked with @tmat and he said the same thing. I've messaged you offline to try and determine if it is an issue with the pool or build definition.